mobx_codegen/lib/src/template/method_override.dart

```diff
@@ -10,31 +10,32 @@ class MethodOverrideTemplate {
 
   MethodOverrideTemplate.fromElement(
     // ignore: deprecated_member_use
-    ExecutableElement method,
+    ExecutableElement2 method,
     LibraryScopedNameFinder typeNameFinder,
   ) {
     // ignore: prefer_function_declarations_over_variables, deprecated_member_use
-    final param = (ParameterElement element) => ParamTemplate(
-        name: element.name,
+    final param = (FormalParameterElement element) => ParamTemplate(
+        name: element.name3!,
         type: typeNameFinder.findParameterTypeName(element),
         defaultValue: element.defaultValueCode,
         hasRequiredKeyword: element.isRequiredNamed);
 
-    final positionalParams = method.parameters
+    final positionalParams = method.formalParameters
```

Comment:
Note the change to `formalParameters` in the migration doc:

https://github.com/dart-lang/sdk/blob/main/pkg/analyzer/doc/element_model_migration_guide.md#formal-parameters